quarkus-opentelemetry-exporter-gcp/integration-tests/src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+# Setting these for tests explicitly. Not required in normal application
+quarkus.application.name=opentelemetry-exporter-gcp-integration-test
+quarkus.application.version=999-SNAPSHOT
+
+simple/mp-rest/url=${test.url}
+
```

Comment:
In the docs you ask to deactivate the default otlp exporter, probably it should be deactivated here as well.